### PR TITLE
fix #671 meta fileversion update

### LINKED ISSUE #671: New FileVersion dont clone FileVersion Meta Title

### LINKED ISSUE #682: Upload new version of media
When i upload a new version of media i get an server-error and the details of the image will be deleted!

### PR DESCRIPTION
**Pull request by alexander-schranz**
__Tasks:__

- [x] test coverage
- [ ] gather feedback for my changes
- [ ] submit changes to the documentation
- [x] added changelog line

Opens new Issue https://github.com/sulu-cmf/sulu/issues/770

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | hopefully
| Fixed tickets | fixes #671, fixes #682
| BC Breaks     | none
| Doc           | none

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Api/Media.php [hunk 1]
```diff
@@ -562,7 +562,7 @@ public function getCreated()
      */
     public function setCreator($creator)
     {
-        $this->entity->setChanger($creator);
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Api/Media.php]: why only creator not both?

**Comment by wachterjohannes** [comment]
@alexander-schranz technically it works but when i upload a new version in the admin ui i have to reload the page ... i think this should reload automatically the list of pictures!

**Comment by wachterjohannes** [comment]
or better it should reload the new thumbnail of the media!

File: src/Sulu/Bundle/MediaBundle/Api/Media.php (hunk 1 above)
Comment (alexander-schranz) [inline src/Sulu/Bundle/MediaBundle/Api/Media.php]: this is the setter function for creator in the API object currently it was false set to changer instead to creator

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Api/Media.php]: ok i see (=

Comment (dantleech) [inline src/Sulu/Bundle/MediaBundle/Api/Media.php]: btw. this week I will probably create an ORM listener to automatically set `created` and `changed` fields (e.g. for all Entities implementing `Timestampable`). Could probably do the same for `creator` and `changer`. /cc @danrot

**Comment by wachterjohannes** [comment]
@alexander-schranz are my comments fixed?

**Comment by dantleech** [comment]
Please explain what the PR does in the description and why it is required.

File: src/Sulu/Bundle/MediaBundle/Media/Manager/DefaultMediaManager.php [hunk 2]
```diff
@@ -597,12 +596,13 @@ protected function createMedia($data, $user)
     }
 
     /**
-     * @param SymfonyFile|null $uploadedFile
-     * @return object
+     * @param SymfonyFile $file
+     * @return int
```
Comment (dantleech) [inline src/Sulu/Bundle/MediaBundle/Media/Manager/DefaultMediaManager.php]: `integer`

**Comment by dantleech** [comment]
I think @wachterjohannes still has an unanswed question.

**Comment by alexander-schranz** [comment]
@dantleech the PR fixes the two issues #671 #682 that the Meta(Title Description) is lost when a new FileVersion is set.
I talked with @wachterjohannes because of the frontend bug this will be not place of this PR because I dont have much time currently for it.

**Pull request merged**